declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,63 @@
+* You have validated that the user-defined cluster-scoped instance is configured with the cluster roles and cluster role bindings for the following components:
++
+** Argo CD Application Controller
+** Argo CD server
+** Argo CD ApplicationSet Controller (provided the ApplicationSet Controller is created)
+* You have xref:../declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc#gitops-disabling-the-creation-of-the-default-cluster-roles-for-the-cluster-scoped-instance_customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances[disabled the creation of the default cluster roles] for the cluster-scoped instance.
+
+// Creating aggregated cluster roles
+include::modules/gitops-creating-aggregated-cluster-roles.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc#gitops-enabling-the-creation-of-aggregated-cluster-roles_customizing-permissions-by-creating-aggregated-cluster-roles[Enabling the creation of aggregated cluster roles]
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc#gitops-creating-user-defined-cluster-roles-and-configuring-user-defined-permissions-for-application-controller_customizing-permissions-by-creating-aggregated-cluster-roles[Creating user-defined cluster roles and configuring user-defined permissions for Application Controller]
+
+// Enabling the creation of aggregated cluster roles
+include::modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+* xref:../argocd_instance/setting-up-argocd-instance.adoc#setting-up-argocd-instance[Installing a user-defined Argo CD instance]
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc#gitops-creating-user-defined-cluster-roles-and-configuring-user-defined-permissions-for-application-controller_customizing-permissions-by-creating-aggregated-cluster-roles[Creating user-defined cluster roles and configuring user-defined permissions for Application Controller]
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc#gitops-creating-aggregated-cluster-roles_customizing-permissions-by-creating-aggregated-cluster-roles[Creating aggregated cluster roles]
+
+// Creating user-defined cluster roles and configuring user-defined permissions for Application Controller
```

Comment:
Suggestion: Can we change the "Enabling the creation of aggregated cluster roles" and "Creating user-defined cluster roles and configuring user-defined permissions for Application Controller" sections to "**leveloffset=+2**" as I feel they are subsets of the "Creating aggregated cluster roles" section?

Reply:
Sorted the conversation with @Dhruv-Soni11. Retaining the content as is.